DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/servlets/PresetServlet.java

```diff
@@ -42,24 +42,27 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
     // parse each parameter from the save preset form.
     String userEmail = request.getParameter("userEmail");
     String userId = request.getParameter("userId");
```

Comment:
Just making sure - are you getting the user id by doing userService.getCurrentUser().getUserId(); ?

Reply:
Yeah, I got the userId from the login status that Erik provided.